auem/train.py

```diff
@@ -1,10 +1,18 @@
+import sys, os
+from dotenv import load_dotenv
+load_dotenv()
+
+sys.path.append(os.environ.get("path_to_auem"))
+
 import hydra
 from omegaconf import DictConfig
 from tqdm import tqdm
+import torch
```

Comment:
you need to run your pre-commits; your imports are not sorted.

Reply:
something's up and I can't run them on my machine. Let's try and debug?